client/app/components/Modal/__tests__/Modal.spec.jsx

```diff
@@ -304,11 +332,11 @@ describe('Modal', () => {
           />
         );
 
-        const wrapper = mount(component);
-        expect(wrapper.find('.avatar').exists()).toEqual(true);
-        wrapper.find('.avatar').simulate('click');
-        expect(wrapper.find('.modalBackdrop').exists()).toEqual(true);
-        expect(wrapper.find('.modal').exists()).toEqual(true);
+        const { container } = render(component);
+        expect(container.querySelector('.avatar')).toBeInTheDocument();
```

Comment:
With testing-library, it's preferable to query the rendered image itself, but in this case I'm having trouble querying the image because the Avatar is also wrapped inside the LazyLoad component.

I tried using `findByRole` which [combines](https://testing-library.com/docs/dom-testing-library/api-queries#findby) `waitFor` and `getByRole`:

```
const image = await screen.findByRole('image', {}, { timeout: 4000 });
expect(image).toBeInTheDocument();
```

Which should work afaik (and I tried it with a longer timeout too), but it wasn't cooperating. Maybe this test can be revisited 👍